tensorflow_probability/python/experimental/mcmc/weighted_resampling_test.py

```diff
@@ -299,6 +299,46 @@ def resample_with_target_distribution(self):
         tf.reduce_sum(tf.nn.softmax(new_log_weights) * new_particles),
         30., atol=1.)
 
+  def test_with_target_distribution_dim_one(self):
+    particles = np.linspace(0., 500., num=2500, dtype=np.float32)
+    stacked_particles = np.stack([particles, particles], axis=0)
+
+    stacked_log_weights = poisson.Poisson(20.).log_prob(stacked_particles)
+
+    # Resample particles to target a Poisson(20.) distribution.
+    new_particles, _, new_log_weights = resample(
+        stacked_particles, stacked_log_weights,
+        resample_fn=resample_systematic,
+        particles_dim=1,
+        seed=test_util.test_seed(sampler_type='stateless'))
+
+    self.assertAllMeansClose(new_particles,
+                             [20., 20.],
+                             axis=1,
+                             atol=1e-2)
+    self.assertAllClose(
```

Comment:
This might be a more clear as:
```
self.assertAllClose(
        tf.reduce_sum(tf.nn.softmax(new_log_weights) * new_particles, axis=1),
        [20., 20.],
        atol=1e-2)
```